src/screens/Learn/Faq/Faq.tsx

```diff
@@ -47,6 +48,7 @@ const Faq: React.FC = () => {
         pageTitle={metadataTitle}
         pageDescription={`${date} ${metadataDescription}`}
       />
+      <FaqStructuredData />
```

Comment:
I think we need to use `Helmet` here (see [AppMetaTags.js#L24](https://github.com/covid-projections/covid-projections/blob/develop/src/components/AppMetaTags/AppMetaTags.js#L24)) to make sure that the script is in the `head` element in the page instead of inside the body.